### PR TITLE
Add solution for the first part

### PR DESCRIPTION
**Pull request by AndreiYazadzhi**

### CODE REVIEW

**Review by tonitrus55 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@
 В конце первой части создать Pull request, и закинуть на проверку.
 
 ### Second HW
-Приступать к виполнению второй части домашки желатильно только после принятия первой, чтоби избежать конфликтов и проблем з git.
+Приступать к виполнению второй части домашки желательно только после принятия первой, чтоби избежать конфликтов и проблем з git.
```
Comment (dmytro-hryhoruk) [inline README.md:18]: 👍

File: src/main/java/Main.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import controller.BetHandler;
+import controller.ConsoleHandler;
+import controller.UserHandler;
+
+public class Main {
+
+    public static void main(String[] args) {
+        ConsoleHandler userHandler = new UserHandler();
```
Comment (dmytro-hryhoruk) [inline src/main/java/Main.java]:
```suggestion
        ConsoleHandler userConsoleHandler = new UserHandler();
```

File: src/main/java/Main.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+        System.out.println("Enter nickname and password");
+        userHandler.handle();
+        ConsoleHandler betHandler = new BetHandler();
```
Comment (dmytro-hryhoruk) [inline src/main/java/Main.java]:
```suggestion
        ConsoleHandler betConsoleHandler = new BetHandler();
```

File: src/main/java/controller/BetHandler.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
+package controller;
+
+import dao.BetDao;
+import dao.impl.BetDaoImpl;
+import java.util.Scanner;
+import model.Bet;
+
+public class BetHandler implements ConsoleHandler {
+    BetDao betDao = new BetDaoImpl();
```
Comment (dmytro-hryhoruk) [inline src/main/java/controller/BetHandler.java]: access modifier

File: src/main/java/controller/BetHandler.java [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+                return;
+            }
+            Bet bet = null;
+            try {
+                String[] betData = command.split(" ");
+                int value = Integer.parseInt(betData[0]);
+                double risk = Double.parseDouble(betData[1]);
+                bet = new Bet(value, risk);
+            } catch (ArrayIndexOutOfBoundsException | NumberFormatException e) {
+                System.out.println("Please,enter correct data");
```
Comment (dmytro-hryhoruk) [inline src/main/java/controller/BetHandler.java:28]: should we add our bet if these exceptions had been thrown?

File: src/main/java/controller/BetHandler.java [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+            }
+            betDao.add(bet);
+            System.out.println(bet == null ? null : bet.toString());
```
Comment (dmytro-hryhoruk) [inline src/main/java/controller/BetHandler.java:31]: Look inside method println

File: src/main/java/controller/BetHandler.java [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+        }
```
Comment (dmytro-hryhoruk) [inline src/main/java/controller/BetHandler.java:32]: Let's print all bets using method getAll()

File: src/main/java/controller/UserHandler.java [hunk 8]
```diff
@@ -0,0 +1,26 @@
+package controller;
+
+import dao.PlayerDao;
+import dao.impl.PlayerDaoImpl;
+import java.util.Scanner;
+import model.Player;
+
+public class UserHandler implements ConsoleHandler {
+    PlayerDao playerDao = new PlayerDaoImpl();
```
Comment (dmytro-hryhoruk) [inline src/main/java/controller/UserHandler.java]: same

File: src/main/java/controller/UserHandler.java [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+
+        String command = scanner.nextLine();
+        try {
+            String[] userData = command.split(" ");
+            String nickname = userData[0];
+            String password = userData[1];
+            playerDao.addPlayer(new Player(nickname, password));
+        } catch (ArrayIndexOutOfBoundsException e) {
+            System.out.println("Please,repeat with correct data");
+            System.exit(1);
```
Comment (dmytro-hryhoruk) [inline src/main/java/controller/UserHandler.java]: how user will repeat if the program will end?

File: src/main/java/controller/UserHandler.java [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+        }
```
Comment (dmytro-hryhoruk) [inline src/main/java/controller/UserHandler.java:29]: let's print all players too

**Review by dmytro-hryhoruk — CHANGES_REQUESTED** [review]

File: src/main/java/controller/BetHandler.java (hunk 5 above)
Comment (AndreiYazadzhi) [reply src/main/java/controller/BetHandler.java:27]: Sry,but i didn`t understand what do you mean

File: src/main/java/controller/BetHandler.java (hunk 6 above)
Comment (AndreiYazadzhi) [reply src/main/java/controller/BetHandler.java:31]: If you mean null check i added it becuase of toString with null value give NPE,not sout cause

Comment (dmytro-hryhoruk) [reply src/main/java/controller/BetHandler.java:31]: sout calling toString() automatically. And also check if the value is null before calling. So you can easily pass any variable to sout without thinking about NPE

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**